docs/configuration/i18n.mdx

```diff
@@ -259,7 +261,10 @@ const field: Field = {
   name: 'myField',
   type: 'text',
   label: (
-    { t }: { t: TFunction<CustomTranslationsKeys | DefaultTranslationKeys> }, // The generic passed to TFunction does not automatically merge the custom translations with the default translations. We need to merge them ourselves here
-  ) => t('fields:addLabel'),
+    { t: defaultT }
+  ) => {
+    const t = defaultT as TFunction<CustomTranslationsKeys>
+    return t('fields:addLabel')
+  },
```

Comment:
This is awful, but it is the only alternative without modifying types like `LabelFunction` and making a breaking change